src/main/java/com/szeloch/jan/parkingsystemmanagement/utils/ParkingTicketsUtil.java

```diff
@@ -0,0 +1,66 @@
+package com.szeloch.jan.parkingsystemmanagement.utils;
+
+import com.szeloch.jan.parkingsystemmanagement.rest.model.ParkingRate;
+import com.szeloch.jan.parkingsystemmanagement.rest.model.ParkingRevenue;
+import com.szeloch.jan.parkingsystemmanagement.rest.model.ParkingTicket;
+import com.szeloch.jan.parkingsystemmanagement.rest.repository.ParkingRatesRepository;
+import com.szeloch.jan.parkingsystemmanagement.rest.repository.ParkingTicketsRepository;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import static java.time.temporal.ChronoUnit.HOURS;
+
+@Component
+public class ParkingTicketsUtil {
```

Comment:
To chyba nie jest util, tylko bardzo biznesowa klasa?

Reply:
Zgadza się, nie ma metod statycznych. Na początku tak ją nazwałem i utworzyłem statyczne metody, ale później to zmieniłem. Nazwa do poprawy.